src/OrchardCore.Modules/OrchardCore.AuditTrail/Views/Admin/Index.cshtml

```diff
@@ -0,0 +1,84 @@
+@model AuditTrailViewModel
+
+@{
+    var auditTrailEvents = Model.AuditTrailEvents;
+
+    var orderByItems = new List<SelectListItem> 
+    {
+        new SelectListItem 
+        {
+            Text = T["Date (newest first)"].Value, 
```

Comment:
Not sure it will work. T is an HTML localizer, so doing .Value will encode it. Then finaly the text will be re-encoded when used in a view. You can do it with a string localizer, or not usin SelectListItem.